docs/v1/open_questions.md

```diff
@@ -0,0 +1,112 @@
+# Open Questions
+
+This document contains open questions regarding the
+[API design document](./design.md).
+
+## 1. Is the `ServiceInstanceBinding` resource necesssary?
+
+The `ServiceInstanceBinding` object is a byproduct of the successful execution
+of a `ServiceInstanceClaim`. Instead of writing a new `ServiceInstanceBinding`
+object, can the service catalog controller just add the binding information
+directly into the `ServiceInstanceClaim` on success?
+
+## Should we write Kubernetes references for binding information?
+
+Currently, the information in a `ServiceInstanceBinding` that points to created
+resources looks like this:
+
+```yaml
+secrets:
+  - my-zip-code-creds-password
+configMaps:
+  - my-zip-code-creds-conn-info
+```
+
+Instead, should we write pointers to these resources using Kubernetes reference
+notation, as below?
+
+```yaml
+resources:
+  - /api/v1/namespaces/${NAMESPACE}/secrets/my-zip-code-creds-password
+  - /api/v1/namespaces/${NAMESPACE}/configmaps/my-zip-code-creds-conn-info
+```
+
+## 2. Should `ServiceInstance`s be namespaced?
+
+If `ServiceInstance`s were namespaced, then operators could enforce their
+security with [RBAC](http://kubernetes.io/docs/admin/authorization/#rbac-mode).
+
+If we do namespace `ServiceInstance`s, however, a few questions arise:
+
+1. It would become messy for pods to share a `ServiceInstance` inter-namespace
+1. A consumer may still submit a `ServiceInstanceClaim` that refers to a
+  `ServiceInstance` that is in a RBAC-forbidden namespace.
+  1. Possible solution: the controller enforces security, as is the case with
+     the current design doc
+
+## 3. Should we keep `ProvisioningPolicy` and `BindingPolicy`?
+
+As mentioned in (2), we may use RBAC to replace the functionality that these
+two policies offered. However, if `ServiceClass` and/or `ServiceInstance`
+remain cluster-global, we'll likely need some way to restrict certain
+claims (likely all claims in a given namespace) from successfully executing
+actions against one or more `ServiceClass`es / `ServiceInstance`s
+
+## 4. Can we reduce the security logic each controller must implement?
+
+As the document stands, service controllers will need to implement logic to
+enforce at least `ProvisioningPolicy` and `BindingPolicy` objects. This logic
+will be non-trivial, redundant if every controller has to implement it, and
+error-prone.
+
+Regardless of whether we keep these policy objects (see (3) above), can we add
+some functionality, a library, or make some other design changes to ensure this
+implementation is easier and less error-prone?
+
+## 5. How will quotas work?
+
+The section entitled "Quotas" is TBD. It needs more thought and detail.
+
+## 6. Should we have a `ServiceController` resource?
```

Comment:
@MHBauer Since this is an open question, I can only give my personal opinion here until we discuss in detail as a group. See inline.

>Where would this be tracked otherwise? K8s service namespace?

IMO these would be cluster-global resources, just as we're planning to have `ServiceClass` at least

>How would an individual controller know to act?

This is controller-dependent. Currently, our prototype controller is configured via environment variables, for example. It's the cluster operator's responsibility to configure controllers properly

>What if more than one thinks it can take an action?

Because these resources have a `ResourceVersion` in them, we can use Kubernetes's optimistic concurrency controls to prevent more than one controller from taking the same action on a claim. I haven't outlined the specific implementation of such concurrency control in this document.